jaxrs-api/src/main/java/jakarta/ws/rs/core/UriInfo.java

```diff
@@ -230,6 +231,70 @@ public interface UriInfo {
      */
     public List<String> getMatchedURIs();
 
+    /**
+     * <p>
+     * Get a template concatenated of all {@link Path Paths} matched by all the resources when matching the current request.
+     * </p>
+     * <p>
+     * Each {@link Path} value used to match a resource class, a sub-resource method or a sub-resource locator is concatenated
+     * to a single {@code String} value. The template does not include query parameters but do include matrix parameters
```

Comment:
...but does include

Reply:
"to a single" -> "into a single"?

Reply:
@jim-krueger what do you mean by "does include"? The URI template itself does not contain the query parameter in the request unless the URI template contains a string containing the `\\?` symbol.

Reply:
Sorry, should have been more clear.   
`but do include matrix parameters`
should be:
`but does include matrix parameters`